lib/Config.ts

```diff
@@ -47,24 +47,28 @@ class Config {
 
   constructor() {
     const platform = os.platform();
-    let lndDefaultDatadir;
+    let lndDefaultDatadir: string;
+    let raidenDefaultKeystorePath: string;
     switch (platform) {
       case 'win32': { // windows
         const homeDir = process.env.LOCALAPPDATA!;
         this.xudir = path.join(homeDir, 'Xud');
         lndDefaultDatadir = path.join(homeDir, 'Lnd');
+        raidenDefaultKeystorePath = path.join(homeDir, 'Ethereum');
         break;
       }
       case 'darwin': { // mac
         const homeDir = process.env.HOME!;
         this.xudir = path.join(homeDir, '.xud');
         lndDefaultDatadir = path.join(homeDir, 'Library', 'Application Support', 'Lnd');
+        raidenDefaultKeystorePath = path.join(homeDir, 'Library', 'Ethereum');
         break;
       }
       default: { // linux
         const homeDir = process.env.HOME!;
         this.xudir = path.join(homeDir, '.xud');
         lndDefaultDatadir = path.join(homeDir, '.lnd');
+        raidenDefaultKeystorePath = path.join(homeDir, '.ethereum');
```

Comment:
Not a big deal, but I'm wondering if the default path to the generated keystore file should be xud's data directory instead. When I was testing the feature without reading the code I assumed it would be there after reading the output from xucli: `The keystore for raiden was initialized`.

Reply:
I just tried to match the default location that raiden uses for the keystore, but if the xud data dir makes more sense then that's fine. What do you think @michael1011  @kilrau ?

Reply:
I think what Daniel did is right. [Geth](https://github.com/ethereum/go-ethereum/wiki/Backup-&-restore), as well as [raiden](https://medium.com/raiden-network/introducing-the-raiden-wizard-6c7c61c5b695) place the keystore file in

    Mac: `~/Library/Ethereum/keystore`
    Linux: `~/.ethereum/keystore`
    Windows: `%APPDATA%\Ethereum\keystore`

by default. What we can do, is adjust the output from xucli: `The keystore for raiden was initialized in [folder].`

Reply:
@reliveyy Docker should place the generated keystore file into the geth folder (we already have a keystore file there which should be replaced (correct me if I'm wrong @sangaman @erkarl @michael1011 )
```
root@K-Yoga:/home/kilrau/.xud-docker/mainnet/data/geth/keystore# ls -la
total 20
drwx------ 2 root root 4096 Sep 25 10:40 .
drwxr-xr-x 4 root root 4096 Sep 25 10:40 ..
-rw------- 1 root root  491 Sep 25 10:40 UTC--2019-09-25T08-40-13.140840485Z--ee9c0a1ed60057e3d0fcd5052aec084edad4b89e
```

Until https://github.com/raiden-network/raiden/issues/4298 is implemented in raiden, the keystore file is used by geth.

Reply:
What if the existing ethereum keystore path/folder already has 10+ keystore files in it. How do you know which one to use?

Reply:
Good point. `xud` should definitely know which one to use in that case. How I see it, we have two options and this needs to be added in this PR:
1. name the created keystore file sth like `xud_keystore` and make `xud` look for this file in the ethereum default path (quick n easy)
2. create a `keystore` config paramter for `[raiden]` in `xud.conf` which saves the full path to the keystore file to be used. This would also allow the user to easily switch to other (existing) keystore files, which will definitely be a requirement in future (better)

Not sure if there's a 3rd option. Let us know what you think @sangaman

Reply:
Just to clarify - unless I'm missing something xud doesn't need to know about raiden's keystore path after `xucli create`. The question  (with the current implementation) is how does raiden know which one to use if it saves it to a path with existing keystore files?

Reply:
>  xud doesn't need to know about raiden's keystore path

Right. But shouldn't `xud` somehow know which of the ethereum 'accounts' it created? To know which one to try to unlock upon `unlock`? Of course we could just try all in path and whichever successfully unlock can be used by raiden/geth.

> The question (with the current implementation) is how does raiden know which one to use if it saves it to a path with existing keystore files?

I asked the raiden team.

Reply:
> > The question (with the current implementation) is how does raiden know which one to use if it saves it to a path with existing keystore files?
> 
> I asked the raiden team.

You *have* to specify `--address` and `--pasword-file` which identifies the keystore file.

Reply:
I was thinking the same config parameter here used to tell xud where to create the keystore would also be used to unlock the keystore once we have that functionality available.

I'm still not sure what the best approach here is in terms of the default value but I still figure it's OK to use the default eth folder. In a case where there are other keystores, we maybe should just expect users to set a path here in the config. It seems to me that either way, they'll need to be configuring either raiden OR xud *unless* we use the default eth keystore pass.

Reply:
Let's leave it as is.